lib/format/duration/formatters/humanized.ex

```diff
@@ -19,12 +19,18 @@ defmodule Timex.Format.Duration.Formatters.Humanized do
 
   ## Examples
 
-      iex> alias Timex.Duration
-      iex> Duration.from_erl({1435, 180354, 590264}) |> #{__MODULE__}.format
+      iex> use Timex
+      ...> Duration.from_erl({1435, 180354, 590264}) |> #{__MODULE__}.format
       "45 years, 6 months, 5 days, 21 hours, 12 minutes, 34 seconds, 590.264 milliseconds"
-      iex> Duration.from_erl({0, 65, 0}) |> #{__MODULE__}.format
+
+      iex> use Timex
+      ...> Duration.from_erl({0, 65, 0}) |> #{__MODULE__}.format
       "1 minute, 5 seconds"
 
+      iex> use Timex
+      ...> Duration.from_erl({0, 1, 1_000_000}) |> #{__MODULE__}.format
+      "2 seconds"
+
```

Comment:
I might suggest reversing the order of these three doctests, so that the simplest case comes first. I think that it would make the documentation a little more educational? Does that make sense?